src/commands/api-mesh/run.js

```diff
@@ -201,9 +202,8 @@ class RunCommand extends Command {
 				if (!portNo) {
 					portNo = 5000;
 				}
-
-				this.log(`Starting server on port : ${portNo}`);
-				await startGraphqlServer(meshId, portNo, flags.debug);
+				meshId = process.env.MESH_ID ?? '000000000000-0000-0000-0000-000000000000';
```

Comment:
Is `MESH_ID` a new field we are introducing? Can we not set it to `mesh` or `testMesh` or `sampleMesh`?